### PR TITLE
added note on advanced shell privs per BZ

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1369513

### CODE REVIEW

**Comment by mburke5678** [comment]
@sferich888 I am not clear on what you mean by "advanced shell" permissions. Privileged SCC? Can you specify what is needed?

**Comment by mburke5678** [comment]
@sferich888 PTAL
Added:
"The *F5 BIG-IQ* user account must have access to the Advanced Shell (Bash) on the BIG-IQ system."

**Comment by mburke5678** [comment]
@bmeng PTAL

**Comment by bmeng** [comment]
@lihongan Please help review the doc.

File: install_config/router/f5_router.adoc [hunk 1]
```diff
@@ -53,6 +53,7 @@ requirements:
 * For edge integration (not recommended):
 ** A working ramp node
 ** A working tunnel to the ramp node
+* An {product-title} user assigned the xref:../../admin_guide/manage_scc.html#listing-security-context-constraints[`privileged` security context constraint]
```
Comment (lihongan) [inline install_config/router/f5_router.adoc]: we always assign `privileged` scc to a service account like `router`, not an user.

File: install_config/router/f5_router.adoc [hunk 2]
```diff
@@ -133,7 +134,8 @@ BIG-IP®* host:
 address.
 
 |`--external-host-username`
-|Specifies the *F5 BIG-IP®* user name (typically *admin*).
+|Specifies the *F5 BIG-IP®* user name (typically *admin*). 
+The *F5 BIG-IQ* user account must have access to the Advanced Shell (Bash) on the BIG-IQ system.
```
Comment (lihongan) [inline install_config/router/f5_router.adoc]: what's the `BIP-IQ` mean? should be `*F5 BIG-IP®*` right ?

File: install_config/router/f5_router.adoc [hunk 3]
```diff
@@ -343,4 +345,9 @@ $ oc adm router \
     --external-host-vxlan-gw=10.131.0.5/14
 ----
 +
+[NOTE]
++
+====
+The `external-host-username` is a *F5 BIG-IQ* user account with access to the Advanced Shell (Bash) on the BIG-IQ system.
```
Comment (lihongan) [inline install_config/router/f5_router.adoc]: same to above, should be `*F5 BIG-IP®*` ?

**Comment by lihongan** [comment]
IMHO, we don't need add the link to a f5 KB article, the user account should be created by F5 BIP-IP admin and F5 admin should have the knowledge about the user account properties (Advance shell).
Actually, when you create a new user via F5 web console, you need click the drop down box called "Terminal Access" and select one of three options: Disabled, **Advanced shell** and tmsh, the second options just we note in this PR.
BTW, the default F5 BIG-IP admin user has `Advanced shell` option as default.

**Comment by mburke5678** [comment]
@lihongan I have made your changes. PTAL

File: install_config/router/f5_router.adoc [hunk 4]
```diff
@@ -133,7 +134,8 @@ BIG-IP®* host:
 address.
 
 |`--external-host-username`
-|Specifies the *F5 BIG-IP®* user name (typically *admin*).
+|Specifies the *F5 BIG-IP®* user name (typically *admin*). 
+The *F5 BIG-IQ* user account must have access to the Advanced Shell (Bash) on the F5 BIG-IP system.
```
Comment (lihongan) [inline install_config/router/f5_router.adoc]: Seems  here ```The *F5 BIG-IQ* user account``` has not be updated to `BIG-IP`. I checked the F5 doc and found that BIG-IQ is Centralized Management system and can discover, manage and monitor remote BIG-IP devices, but it is not needed for OpenShift and F5 integration, so I think better to use `BIP-IP` here avoiding confusion.

File: install_config/router/f5_router.adoc [hunk 5]
```diff
@@ -343,4 +345,9 @@ $ oc adm router \
     --external-host-vxlan-gw=10.131.0.5/14
 ----
 +
+[NOTE]
++
+====
+The `external-host-username` is a *F5 BIG-IQ* user account with access to the Advanced Shell (Bash) on the F5 BIG-IP system.
```
Comment (lihongan) [inline install_config/router/f5_router.adoc]: same to above, ```a *F5 BIG-IQ* user account```

File: install_config/router/f5_router.adoc (hunk 4 above)
Comment (mburke5678) [reply install_config/router/f5_router.adoc]: @lihongan Thank you for clarifying. I don't think I saw that I was using the wrong term.

**Comment by mburke5678** [comment]
@sferich888 The BZ was filed against 3.2; do we want to port this change all the way back to 3.2?

**Comment by mburke5678** [comment]
[rev_history]
|xref:../install_config/router/f5_router.adoc#install-config-router-f5[Installation -> Using the F5 Router Plug-in]
|Added a note that the F5 user needs Advanced Shell permission to use the xref:../install_config/router/f5_router.adoc#deploying-the-f5-router[F5 Router Plug-in].
%

**Comment by mburke5678** [comment]
@openshift/team-documentation PTAL

File: install_config/router/f5_router.adoc [hunk 6]
```diff
@@ -343,4 +345,9 @@ $ oc adm router \
     --external-host-vxlan-gw=10.131.0.5/14
 ----
 +
+[NOTE]
++
+====
+The `external-host-username` is a *F5 BIG-IP* user account with access to the Advanced Shell (Bash) on the F5 BIG-IP system.
+
```
Comment (bfallonf) [inline install_config/router/f5_router.adoc]: The markup above is a little off, and the sentence below has been pulled into the note box. line 352 should have another set of =s and perhaps no line space between the NOTE heading and the text. See:

https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#admonitions

**Comment by bfallonf** [comment]
@mburke5678 A comment from me, but looks good otherwise.

**Comment by lihongan** [comment]
LGTM, thank you.

File: install_config/router/f5_router.adoc (hunk 6 above)
Comment (mburke5678) [reply install_config/router/f5_router.adoc]: @bfallonf Fixed. Thanks!

**Pull request merged**